cmd/issuer_initializer/main.go

```diff
@@ -102,14 +107,29 @@ func main() {
 	// services initialization
 	mtService := services.NewIdentityMerkleTrees(mtRepository)
 
-	revocationSettings := services.CredentialRevocationSettings{
-		RHSEnabled:        cfg.ReverseHashService.Enabled,
-		RHSUrl:            cfg.ReverseHashService.URL,
-		Host:              cfg.ServerUrl,
-		AgentIden3Enabled: false,
+	commonClient, err := ethclient.Dial(cfg.Ethereum.URL)
+	if err != nil {
+		log.Info(ctx, "the ethereum rcp is", "url", cfg.Ethereum.URL)
+		log.Error(ctx, "error dialing with ethclient", "err", err)
+		panic("Error dialing with ethclient: " + err.Error())
```

Comment:
I would just `return` as we do in the cases when it fails

Reply:
Agree.. Just return